src/features/singleRight/delegate/ChooseServicePage/ChooseServicePage.tsx

```diff
@@ -119,6 +112,39 @@ export const ChooseServicePage = () => {
             >
               {t('common.proceed')}
             </Button>
+            <Popover
```

Comment:
Also, also:
I would expect to be able to close the popover when clicking outside of it or by clicking on the trigger button a second time, but here, I can't. It also can't be closed with esc, which breaks accessibility demands. The only way to close it is by hitting the button inside the popover.

It seems to work with the popover on the Rights page though, so I'm not sure why its not working here 🤔

Reply:
Fixed